devolutions-gateway/src/rdp_proxy.rs

```diff
@@ -344,15 +404,22 @@ where
         security_protocol,
         ironrdp_connector::ServerName::new(server_name),
         server_public_key,
+        // We do not need to specify the Kerberos config here: the sspi-rs can automatically resolve the KDC host via DNS and/or env variable.
```

Comment:
note: I’m pretty sure we’re not going to use the env variables. Do you think the DNS approach is enough?

Reply:
> Do you think the DNS approach is enough?

I improved the `KerberosServr` structure: I added an ability to configure the KDC address